### PR TITLE
Added Sorting the elements of a stack in C

### PR DESCRIPTION
**Pull request by Tanya170620**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

This program sorts the elements of the stack without using another user defined stack. It is a menu driven program where the user first pushes the elements into the stack and then sorts the elements of the stack

### Checklist

- [Yes] I've read the contribution guidelines.
- [Yes] I've checked the issue list before deciding what to submit.
- [Yes] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Sorting the elements of a stack in C #2596

### CODE REVIEW

File: C/ds/stack_sort.c [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+The stack is: 2 8 3 5
+Do you want to continue : y
+
+Enter an option:
+1.push()        2.pop()
+3.display()     4.sort()
+Enter your choice: 4
+The sorted stack is : 2 3 5 8
+Do you want to continue : n
+*/
```
Comment (HarshCasper) [inline C/ds/stack_sort.c]: Add a new line at the EOF

File: C/ds/stack_sort.c [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+void main()
+{
+	struct stack *s = (struct stack *) malloc(sizeof(struct stack));
+	printf("Enter the size of stack : ");
+	scanf("%d", &s->size);
+	s->arr = (int*) malloc((s->size) *(sizeof(int)));
+	s->top = -1;
+	int op;
+	char c;
+	do {
```
Comment (HarshCasper) [inline C/ds/stack_sort.c]: Use a while loop instead of do while.

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Comment by Tanya170620** [comment]
I have made the requested changes. Please check. @HarshCasper

**Review by HarshCasper — APPROVED** [review]

**Comment by HarshCasper** [comment]
@all-contributors bot please add @Tanya170620 for code

**Pull request merged**